.github/scripts/get-version-tags.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+current_tag=$(git tag --list --merged origin/release --sort=-creatordate | head -n 1)
+main_tag=$(git tag --list --merged main --sort=-creatordate | head -n 1)
+
+# Remove the prefix `v` and `-kong-*` suffix for comparison
+released_tag="${current_tag%-kong-*}"
+currentVersion="${released_tag#v}"
+
+# Main branch won't have -kong-* suffix
+newVersion="${main_tag#v}"
+
+echo "Current release tag prefix: $released_tag"
+echo "Upstream tag: $main_tag"
+new_tag="${main_tag}-kong-1"
+if [[ $current_tag != $new_tag ]]; then
+  echo "New tag: $new_tag"
+else
+  echo "Tags are equal, no need to release"
+  exit 0
+fi
+
+# Convert versions to arrays
+IFS='.' read -r -a currentParts <<< "$currentVersion"
+IFS='.' read -r -a newParts <<< "$newVersion"
+
+echo "released_tag=$released_tag" >> $GITHUB_OUTPUT
+echo "main_tag=$main_tag" >> $GITHUB_OUTPUT
+
+echo "Current base tag: $currentVersion"
+echo "Upstream tag: $newVersion"
+
+# Compare each part
+for i in 0 1 2; do
```

Comment:
why do I have this feeling that I've already seen this

Reply:
it's already in main https://github.com/kumahq/go-control-plane/blob/main/.github/scripts/get-version-tags.sh - so why this is showing up?

Reply:
I think it's because of rebase going on the main branch